From 3b3127923708c4fca727a014841fb1494e5eb2ed Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 21 Sep 2020 17:22:59 +0800
Subject: [PATCH] clk: rockchip: rk3568: Fix gcc warning

drivers/clk/rockchip/clk-rk3568.c:162:2: warning: this decimal constant is unsigned only in ISO C90
  RK3568_CPUCLK_RATE(2208000000, 1, 1, 1, 9, 9, 9, 9),

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I697e4b2a9611cf49233d40361863cc820f20d1fa
---
 drivers/clk/rockchip/clk-rk3568.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/rockchip/clk-rk3568.c b/drivers/clk/rockchip/clk-rk3568.c
index d2daf0dc6943..0f97d9ee8062 100644
--- a/drivers/clk/rockchip/clk-rk3568.c
+++ b/drivers/clk/rockchip/clk-rk3568.c
@@ -148,7 +148,7 @@ static struct rockchip_pll_rate_table rk3568_pll_rates[] = {
 
 #define RK3568_CPUCLK_RATE(_prate, _apllcore, _sclk, _acore, _atcore, _gicclk, _pclk, _periph) \
 {								\
-	.prate = _prate,					\
+	.prate = _prate##U,					\
 	.divs = {						\
 		RK3568_CLKSEL0(_apllcore),			\
 		RK3568_CLKSEL1(_sclk),				\
-- 
2.35.3

